doc/release_instructions.md

```diff
@@ -6,6 +6,8 @@ This is because of a dependency on CGO via [this bug](https://github.com/opencon
 
 (We don't want to enable CGO in part because it doesn't play nice w OSX for us)
 
+If you're doing a custom release (off a branch that isn't master, skip to the section at the bottom)
```

Comment:
I think the close-bracket should be before the comma here, right? Also, might as well add a link:

```
If you're doing a custom release (off a branch that isn't master), skip to [the section at the bottom](#custom-release)
```